CMakeLists.txt

```diff
@@ -51,8 +52,22 @@ if(${enable_cbmc_tests})
     enable_testing()
 endif()
 
+option(use_smt_switch "use generic smt interface")
+if(use_smt_switch)
+  download_project(
+    PROJ smt-switch
+    GIT_REPOSITORY  https://github.com/thk123/smt-switch.git
+    GIT_TAG patch-1
```

Comment:
This resolves a `-pedantic` error that is currently got by downloading the master or last release version. I have a PR to fix this in trunk: makaimann/smt-switch#43

If that doesn't get merged soon, the alternative would be to disable `-pedantic` for this project (hopefully @hannes-steffenhagen-diffblue knows a way to do this)